packages/fondue/src/components/Button/Button.tsx

```diff
@@ -122,6 +124,7 @@ const ButtonComponent: ForwardRefRenderFunction<HTMLButtonElement | null, Button
             disabled={disabled}
             form={formId}
             title={title}
+            tabIndex={tabIndex}
```

Comment:
is this change needed because you have to disable the element (`tabindex="-1"`) or do you want to set a custom tabbable order?

Reply:
I want to set a custom tabbable order

Reply:
Ideally we would have this prop on every interactive component. @LidijaStoilova perhaps you have thoughts on this as well?

Separate issue, but there are also a few Fondue form elements that require two `tab` key presses to bypass in the tab order

Reply:
>Separate issue, but there are also a few Fondue form elements that require two tab key presses to bypass in the tab order

Yeah, that sounds like a bug. @mikeyrayvon  Do know which components so we can create ticket for our backlog?

Reply:
Agree with @floroz. Unfortunately I'm not able to access the deploy preview, but happy to take a look at it together @mikeyrayvon.